helm/charts/determined/values-mlde.yaml

```diff
@@ -0,0 +1,280 @@
+# The image registry to use for main image. Defaults to the determinedai repository in DockerHub.
+imageRegistry: hub.myenterpriselicense.hpe.com/hpe-mlde/<SKU>
+
```

Comment:
1. it seems like we're going to allow configuring all images except for determined-master and postgres. why stop here? do we need to allow configuring these as well?
2. with #1 change, `imageRegistry` key may become unused and can probably be removed, or be used to popular `defaultImages`?
3. is this `<SKU>` actually a real URL?.. or uses are supposed to populate it somehow?
4. this `values-mlde.yaml` only seems to change two fields: `imageRegistry` and `enterpriseEdition`. it'll be annoying to maintain: eventually someone will update values.yaml but forget this one. are we sure we need this? or can we keep MLDE options as commented-out lines in `values.yaml`?